src/test/java/MainTest.java

```diff
@@ -1,12 +1,50 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import jakarta.xml.bind.JAXBContext;
+import jakarta.xml.bind.JAXBException;
+import java.nio.file.Path;
+import java.util.Objects;
 import no.sikt.nva.Main;
+import no.sikt.nva.model.DcValue;
+import no.sikt.nva.model.DublinCore;
+import no.sikt.nva.model.Element;
+import nva.commons.core.ioutils.IoUtils;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 public class MainTest {
 
+    private Main main;
+    private String[] arguments;
+
+    @BeforeEach
+    void init() {
+        this.main = new Main();
+        this.arguments = new String[]{};
+    }
+
     @Test
     void runningMain() {
-        var main = new Main();
-        var arguments = new String[]{};
-        main.main(arguments);
+        Main.main(arguments);
+    }
+
+    @Test
+    void shouldBeAbleToConvertDublinCoreXml() throws JAXBException {
+        var dublinCoreXML = IoUtils.inputStreamFromResources(Path.of("dublin_core.xml"));
+        JAXBContext context = JAXBContext.newInstance(DublinCore.class);
+        var dublinCore = (DublinCore) context.createUnmarshaller().unmarshal(dublinCoreXML);
+        var expectedTitle = "Studie av friluftsliv blant barn og unge i Oslo: Sosial ulikhet og sosial utjevning";
```

Comment:
Hadde kanskje endret til noe slikt, tenker på lesbarheten: 

    private DublinCore getDublinCore(String filePath) throws JAXBException {
        var dublinCoreXML = IoUtils.inputStreamFromResources(Path.of(filePath));
        JAXBContext context = JAXBContext.newInstance(DublinCore.class);
        return (DublinCore) context.createUnmarshaller().unmarshal(dublinCoreXML);
    }

Reply:
private DcValue getTitle(DublinCore dublinCore) {
        return dublinCore.getDcValues().stream().filter(dcValue -> isTitle(dcValue)).findFirst().orElseThrow();
    }

Reply:
slik at testen blir enklere å forstå: 

    @Test
    void shouldBeAbleToConvertDublinCoreXml() throws JAXBException {
        DublinCore dublinCore = getDublinCore("dublin_core.xml");
        var expectedTitle = "Studie av friluftsliv blant barn og unge i Oslo: Sosial ulikhet og sosial utjevning";
        var actualTitle = getTitle(dublinCore);
        assertTrue(dublinCore instanceof DublinCore);
        assertEquals(expectedTitle, actualTitle.getValue());
    }